packages/cactus-plugin-ledger-connector-besu/README.md

```diff
@@ -80,6 +81,45 @@ enum Web3SigningCredentialType {
 ```
 > Extensive documentation and examples in the [readthedocs](https://readthedocs.org/projects/hyperledger-cactus/) (WIP) 
 
+## Prometheus Exporter
+
+This class creates a prometheus exporter, which scraps the transactions (total transaction count) for the use cases incorporating the use of Besu connector plugin.
```

Comment:
@jagpreetsinghsasan  Is this paragraph a duplicate of the one I made the typo comment on above?